openapi.yaml

```diff
@@ -224,6 +241,33 @@ paths:
           $ref: '#/responses/BadContent'
         default:
           $ref: '#/responses/InternalServerError'
+  
+  /api/v1/tsr:
+    post:
+      # TODO: Add uploads to transparency log.
+      summary: Returns a timestamp response generated by Rekor
+      operationId: getTimestampResponse
+      tags:
+        - timestamp
+      consumes:
+        - application/json
```

Comment:
Is there a reason to wrap this in JSON vs just sending as `application/timestamp-query` and receiving `timestamp-reply` per the RFC? That would allow us to better integrate with tools like jarsigner.

Reply:
we could then have the `application/json` type be the non-ASN1 format that would rather folks use when implemented.

Reply:
Request yes, I'll do that.
The reply seemed to conflict if rekor uploads to the tlog by default, we'd be sending back some extra info with the response

Reply:
I think writing the RFC-compliant output to a file makes sense, and then just including the rekor information via stdout works too. You could mimic the CreateEntry behavior and return the URL to the entry in the Location HTTP response header without changing the body.

Reply:
Ooooh that's a great idea -- I'll make this change, and update the TODOs about the uploading parts